lib/models/Ride.dart

```diff
@@ -309,12 +323,14 @@ class RideCard extends StatelessWidget {
       {@required this.showConfirmation,
       @required this.showCallDriver,
       @required this.showArrow,
+      @required this.isPast,
       this.parentRideID});
 
   final Ride ride;
   final bool showConfirmation;
   final bool showCallDriver;
   final bool showArrow;
+  final bool isPast;
```

Comment:
I think you might be able to just use ride.type which will be 'active' 'unscheduled' or 'past' so the isPast wouldn't be necessary?